releasenotes/notes/improve_reducenode_propagate-032756e0488d649f.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
```

Comment:
In this case, because there hasn't been a deploy yet that includes the new `ReduceNode`, we don't need a release note.